File: ncarg2d/src/libncarg/stitle/slgeti.f
C
C $Id: slgeti.f,v 1.6 2008-07-27 00:17:26 haley Exp $
C                                                                      
C                Copyright (C)  2000
C        University Corporation for Atmospheric Research
C                All Rights Reserved
C
C The use of this Software is governed by a License Agreement.
C
      SUBROUTINE SLGETI (PNAM,IVAL)
C
C Get in IVAL the integer value of the STITLE parameter named PNAM.
C
        CHARACTER*(*) PNAM
C
C Check for an uncleared prior error.
C
        IF (ICFELL('SLGETI - UNCLEARED PRIOR ERROR',1).NE.0) RETURN
C
C Call SLGETR to obtain the real value of the parameter and then
C return the integer portion of that.
C
        CALL SLGETR (PNAM,RVAL)
        IF (ICFELL('SLGETI',2).NE.0) RETURN
        IVAL=INT(RVAL)
C
C Done.
C
        RETURN
C
      END

File: ncarg2d/man/scrolled_title/slgeti.m
.\"
.\"	$Id: slgeti.m,v 1.1 1993-03-11 16:30:10 haley Exp $
.\"
.TH SLGETI 3NCARG "March 1993" UNIX "NCAR GRAPHICS"
.SH NAME
SLGETI - retrieves specified integer valued parameters.
.SH SYNOPSIS
CALL SLGETI (PNAM, IVAL)
.SH C-BINDING SYNOPSIS
#include <ncarg/ncargC.h>
.sp
void c_slgeti (char *pnam, int *ival)
.SH DESCRIPTION 
.IP PNAM 12
The parameter name, of type character (for example,	'ALN').
.IP IVAL 12
A integer variable in which the current value of the parameter is 
to be returned.
.SH C-BINDING DESCRIPTION
The C-binding argument descriptions are the same as the Fortran 
argument descriptions.
.SH ACCESS
To use SLGETI load the NCAR Graphics libraries ncarg, ncarg_gks,
and ncarg_loc, preferably in that order.  To use c_slgeti load 
the NCAR Graphics libraries ncargC, ncarg_gksC, ncarg, ncarg_gks,
and ncarg_loc, preferably in that order.
.SH SEE ALSO
Online: 
ftitle movies slgeti slgetr slseti slsetr stitle ncarg_cbind
.sp
Hardcopy:
"NCAR Graphics User's Guide, Version 2.00"
.SH COPYRIGHT
(c) Copyright 1987, 1988, 1989, 1991, 1993 University Corporation
for Atmospheric Research
.br
All Rights Reserved
